build/configs/artik053/Make.defs

```diff
@@ -111,12 +111,21 @@ endif
 
 ARCHCFLAGS = -fno-builtin -mcpu=cortex-r4 -mfpu=vfpv3
 ARCHCXXFLAGS = -fno-builtin -mcpu=cortex-r4 -mfpu=vfpv3
+ifneq ($(CONFIG_LIBCXX_HAVE_LIBSUPCXX),y)
+ARCHCXXFLAGS += -fexceptions
+endif
 ifeq ($(QUICKBUILD),y)
 ARCHWARNINGS = -Wall -Werror -Wstrict-prototypes -Wshadow -Wundef -Wno-implicit-function-declaration -Wno-unused-function -Wno-unused-but-set-variable
 ARCHWARNINGSXX = -Wall -Wshadow -Wundef
+ifneq ($(CONFIG_LIBCXX_HAVE_LIBSUPCXX),y)
+ARCHWARNINGSXX += -Werror
+endif
 else
 ARCHWARNINGS = -Wall -Werror -Wstrict-prototypes -Wshadow -Wundef -Wno-implicit-function-declaration -Wno-unused-function -Wno-unused-but-set-variable
-ARCHWARNINGSXX = -Wall -Wshadow -Wundef 
+ARCHWARNINGSXX = -Wall -Wshadow -Wundef
+ifneq ($(CONFIG_LIBCXX_HAVE_LIBSUPCXX),y)
+ARCHWARNINGSXX += -Werror
+endif
```

Comment:
Duplicated at both cases
```
if (quickbuild)
...
ifneq ($(CONFIG_LIBCXX_HAVE_LIBSUPCXX),y)
ARCHWARNINGSXX += -Werror
endif
...
else
...
ifneq ($(CONFIG_LIBCXX_HAVE_LIBSUPCXX),y)
ARCHWARNINGSXX += -Werror
endif
...
```
Let's locate it at out of the conditional for single definition.
endif